frontend/src/components/developerEndpoint.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import axios from 'axios'
+import { Table, Container } from 'react-bootstrap';
+
+export interface Developer {
+    id: number;
+    first_name:string;
+    last_name:string;
+}
+const FetchDevelopers = () => {
```

Comment:
While I love a good ES6, I specifically picked AirBnB style guide as your linter to give you a challenge, force you to learn new things (like configuring exceptions to rules), but also have consistency as a team of developers, since school has you used to working individually. 

The AirBNB Styleguide wants you to declare this as `function FetchDevelopers() {` 

Either update the rule as an exception, enforcing the ES6 style or correct appropriately :)

Reply:
Added some rules to eslint to allow es6 arrow functions. I love my es6 and would keep it that way.